backend/tests/apps/sitemaps/sitemaps_test.py

```
@@ -0,0 +1,193 @@
+    def test_changefreq(self):
+        for item in sm.STATIC_ROUTES:
+            assert self.sitemap.changefreq(item) == item["changefreq"]
+
+    def test_priority(self):
+        for item in sm.STATIC_ROUTES:
+            assert self.sitemap.priority(item) == item["priority"]
+
+
+class TestGetStaticPriority(unittest.TestCase):
+    def test_priority_for_known_path(self):
+        self.assertAlmostEqual(sm.get_static_priority("/projects"), 0.9, places=7)
```

Ruff (0.12.2)

Use a regular `assert` instead of unittest-style `assertAlmostEqual`

(PT009)